spec/factories/completed_lessons.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :completed_lesson do
+    lesson_id {1}
+    user_id {1}
+    completed {true}
```

Comment:
The test is passing because you set `completed: true` on your factory, not because of `CompletedLesson#set_default_to_completed`